bin/collator/src/local/service.rs

```diff
@@ -489,7 +501,7 @@ pub fn start_node(config: Configuration) -> Result<TaskManager, ServiceError> {
             spawn_handle: task_manager.spawn_handle(),
             import_queue,
             block_announce_validator_builder: None,
-            warp_sync_params: None,
+            warp_sync_params: Some(WarpSyncParams::WithProvider(warp_sync)),
```

Comment:
Imo it’s a bit useless to have warp sync on local developer node.

Reply:
totally agree. will revert to None